superset/views/core.py

```diff
@@ -702,7 +702,8 @@ def explore(  # pylint: disable=too-many-locals,too-many-return-statements
             datasource_id, datasource_type = get_datasource_info(
                 datasource_id, datasource_type, form_data
             )
-        except SupersetException:
+        except SupersetException as e:
+            flash(e.message, "danger")
```

Comment:
This seems like a good pattern in general :)

Reply:
Thanks @nytai ! :) Unfortunately I updated message after @junlincc [comment](https://github.com/apache/superset/pull/12468#issuecomment-758899747). Maybe we should change name of variables if those are now `datasets`? WDYT?